CharacterCreation/Class-Specific Documentation/Engineer Processes.txt

```diff
@@ -162,16 +175,16 @@ Action:		1 Action
 Duration:	8 Turns
 Range:		Touch
 
-    You temporarily disable your cybernetic ally's regression system,
+    You temporarily disable your cybernetic ally's regression process,
 the part of their personality program that slows down to triple check the 
 results of their actions. This sends them into a rage mode. They still know
 friend from foe, and they still have a jenky understanding of their higher-
 order skills like lockpicking, hacking, or social finesse, but they have no
 fear, no pain, no remorse and no hesitation for about half of a minute. Your
 ally Cannot feel shock during this duration, and cannot be repaired (they
-won't stand still). They get a bonus +15% to accuracy nearer than 41m. They
+won't stand still). They get a bonus +3 to accuracy nearer than 41m. They
```

Comment:
Nonblocking but 15% on the d100 is equivalent to 1.5 on the d10, so was the effective doubling to 3 of this bonus intended?

Reply:
It was an intentional buff